components/brave_wallet/browser/pref_names.cc

```diff
@@ -27,6 +27,7 @@ const char kSupportEip1559OnLocalhostChain[] =
     "brave.wallet.support_eip1559_on_localhost_chain";
 
 // DEPRECATED
+const char kBraveWalletHardwareKeyring[] = "brave.wallet.keyrings.hardware";
```

Comment:
This should be removed. We don't need it in this use case.

Reply:
done